jwt/decode-jwt.js

```diff
@@ -0,0 +1,25 @@
+"use strict";
+
+require('dotenv').config({path: __dirname + '/../.env'});
+
+const express = require('express')
+const jwt = require('jsonwebtoken')
+
+const app = express();
+
+const signature_secret = VONAGE_SIGNATURE_SECRET;
+
+app.set('port', (process.env.PORT || 5000));
+
+app.get('/', function (req, res) {
+    try {
+        let auth = jwt.verify(req.headers['authorization'], signature_secret)
```

Comment:
Forgot that the auth header is `Bearer <token>`, so need to split that string on the space and use just the token portion. Should be good after this.

```suggestion
        let auth = jwt.verify(req.headers['authorization'].split(' ')[1], signature_secret)
```